DictionaryTypes.txt

```diff
@@ -295,16 +297,29 @@ enum(coil) "Sensor coil type"
     kit_ref_mag	       6002  "KIT reference magnetometer"
     baby_grad	       7001  "Baby-SQUID gradiometer"
     baby_mag	       7002  "Baby-SQUID magnetometer"
+    baby_ref_mag       7003  "Baby-SQUID reference magnetometer"
+    baby_ref_mag2      7003  "Baby-SQUID reference magnetometer 2"
+    artemis123_grad    7501  "Artemis123 gradiometer"
+    artemis123_ref_mag 7502  "Artemis123 reference magnetometer"
+    artemis123_ref_grad 7503  "Artemis123 reference gradiometer"
+    kriss_grad         8001  "KRISS gradiometer"
+    sample_tms_planar  9001  "Sample TMS sensor"
```

Comment:
Same -- MNE has these 7 `FIFFV_COIL_` types that `MEGIN` does not have. Okay to add these? I think they have already been used in `MNE-C` software by @mshamalainen

Reply:
7003 conflicts with LorenzE suggestion. What is the difference between baby_ref_mag (7003) and baby_ref_mag2 (7003)?

Reply:
I think how names are better/more complete. I'll see if he can push those changes here, and then reviewing again should be possible / hopefully not so bad.